.github/workflows/main.yml

```diff
@@ -142,6 +143,14 @@ jobs:
       matrix:
         os: [ 'ubuntu-latest' ]
         python-version: [ "3.10", "3.11", "3.12" ]
+        julia: [ false ]
```

Comment:
@Zeitsperre  Could you look at the changes in the config files to make sure that they are still fine?

Reply:
I have some time tomorrow. Will do.